### PR TITLE
ss/DCOS-21390 Correcting error in restore s3 instructions.

### PR DESCRIPTION
**Pull request by SarahStegall**
## Description
https://jira.mesosphere.com/browse/DCOS-21390
On the Cassandra documentation:

https://docs.mesosphere.com/services/cassandra/2.0.3-3.0.14/disaster-recovery/

On the section to restore an S3 backup, it gives you the command to backup it.

 Expected:

Command should be restore-s3 and not backup-s3

Corrected as described. Affects versions

2.0.1-3.0.14

2.0.2-3.0.14

2.0.3-3.0.14

2.1.0-3.0.16

Previous and subsequent versions do not have this error.

### CODE REVIEW

File: pages/services/cassandra/2.0.1-3.0.14/disaster-recovery/index.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ AWS_ACCESS_KEY_ID=<my_access_key_id>
 AWS_SECRET_ACCESS_KEY=<my_secret_access_key>
 AWS_REGION=us-west-2
 S3_BUCKET_NAME=backups
-dcos cassandra plan start backup-s3 -p SNAPSHOT_NAME=$SNAPSHOT_NAME -p "CASSANDRA_KEYSPACES=$CASSANDRA_KEYSPACES" -p AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -p AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -p AWS_REGION=$AWS_REGION -p S3_BUCKET_NAME=$S3_BUCKET_NAME
+dcos cassandra plan start restore-s3 -p SNAPSHOT_NAME=$SNAPSHOT_NAME -p "CASSANDRA_KEYSPACES=$CASSANDRA_KEYSPACES" -p AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -p AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -p AWS_REGION=$AWS_REGION -p S3_BUCKET_NAME=$S3_BUCKET_NAME
```
Comment (Fabs) [inline pages/services/cassandra/2.0.1-3.0.14/disaster-recovery/index.md]: I guess this is the backup section, here it should still be backup-s3.

File: pages/services/cassandra/2.0.1-3.0.14/disaster-recovery/index.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ AWS_ACCESS_KEY_ID=<my_access_key_id>
 AWS_SECRET_ACCESS_KEY=<my_secret_access_key>
 AWS_REGION=us-west-2
 S3_BUCKET_NAME=backups
-dcos cassandra plan start backup-s3 "SNAPSHOT_NAME=$SNAPSHOT_NAME,AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID,AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY,AWS_REGION=$AWS_REGION,S3_BUCKET_NAME=$S3_BUCKET_NAME"
+dcos cassandra plan start restore-s3 "SNAPSHOT_NAME=$SNAPSHOT_NAME,AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID,AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY,AWS_REGION=$AWS_REGION,S3_BUCKET_NAME=$S3_BUCKET_NAME"
```
Comment (Fabs) [inline pages/services/cassandra/2.0.1-3.0.14/disaster-recovery/index.md]: This is correctly stated as restore.

File: pages/services/cassandra/2.0.2-3.0.14/disaster-recovery/index.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ AWS_ACCESS_KEY_ID=<my_access_key_id>
 AWS_SECRET_ACCESS_KEY=<my_secret_access_key>
 AWS_REGION=us-west-2
 S3_BUCKET_NAME=backups
-dcos cassandra plan start backup-s3 -p SNAPSHOT_NAME=$SNAPSHOT_NAME -p "CASSANDRA_KEYSPACES=$CASSANDRA_KEYSPACES" -p AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -p AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -p AWS_REGION=$AWS_REGION -p S3_BUCKET_NAME=$S3_BUCKET_NAME
+dcos cassandra plan start restore-s3 -p SNAPSHOT_NAME=$SNAPSHOT_NAME -p "CASSANDRA_KEYSPACES=$CASSANDRA_KEYSPACES" -p AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -p AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -p AWS_REGION=$AWS_REGION -p S3_BUCKET_NAME=$S3_BUCKET_NAME
```
Comment (Fabs) [inline pages/services/cassandra/2.0.2-3.0.14/disaster-recovery/index.md]: Same issue here, on the backup section it should keep backup-s3.

File: pages/services/cassandra/2.0.3-3.0.14/disaster-recovery/index.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ AWS_ACCESS_KEY_ID=<my_access_key_id>
 AWS_SECRET_ACCESS_KEY=<my_secret_access_key>
 AWS_REGION=us-west-2
 S3_BUCKET_NAME=backups
-dcos cassandra plan start backup-s3 -p SNAPSHOT_NAME=$SNAPSHOT_NAME -p "CASSANDRA_KEYSPACES=$CASSANDRA_KEYSPACES" -p AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -p AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -p AWS_REGION=$AWS_REGION -p S3_BUCKET_NAME=$S3_BUCKET_NAME
+dcos cassandra plan start restore-s3 -p SNAPSHOT_NAME=$SNAPSHOT_NAME -p "CASSANDRA_KEYSPACES=$CASSANDRA_KEYSPACES" -p AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -p AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -p AWS_REGION=$AWS_REGION -p S3_BUCKET_NAME=$S3_BUCKET_NAME
```
Comment (Fabs) [inline pages/services/cassandra/2.0.3-3.0.14/disaster-recovery/index.md]: Same case as the two above.

File: pages/services/cassandra/2.0.1-3.0.14/disaster-recovery/index.md (hunk 1 above)
Comment (SarahStegall) [reply pages/services/cassandra/2.0.1-3.0.14/disaster-recovery/index.md]: Fixed

File: pages/services/cassandra/2.0.2-3.0.14/disaster-recovery/index.md (hunk 3 above)
Comment (SarahStegall) [reply pages/services/cassandra/2.0.2-3.0.14/disaster-recovery/index.md]: Fixed

File: pages/services/cassandra/2.0.3-3.0.14/disaster-recovery/index.md (hunk 4 above)
Comment (SarahStegall) [reply pages/services/cassandra/2.0.3-3.0.14/disaster-recovery/index.md]: Fixed

**Pull request merged**